charts/enterprise-metrics/templates/graphite-write-proxy-dep.yaml

```diff
@@ -0,0 +1,136 @@
+{{- if .Values.graphite.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "enterprise-metrics.fullname" . }}-graphite-write-proxy
+  labels:
+    app: {{ template "enterprise-metrics.name" . }}-graphite-write-proxy
+    chart: {{ template "enterprise-metrics.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    {{- toYaml .Values.graphite_write_proxy.annotations | nindent 4 }}
+spec:
+  replicas: {{ .Values.graphite_write_proxy.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "enterprise-metrics.name" . }}-graphite-write-proxy
+      release: {{ .Release.Name }}
+  strategy:
+    {{- toYaml .Values.graphite_write_proxy.strategy | nindent 4 }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "enterprise-metrics.name" . }}-graphite-write-proxy
+        # The name label is important for cortex-mixin compatibility which expects certain names for services.
+        name: graphite-querier
+        gossip_ring_member: "true"
```

Comment:
same like with the graphite-querier. I'm not sure what this label means, but it sounds wrong because i don't think the graphite write proxy should have interactions with any ring, no?

Reply:
:+1:  Addressed in https://github.com/grafana/helm-charts/pull/872/commits/6e7f1720ca574756febfc540dede43ed202fe22e